src/twisted/topfiles/8745.bugfix

```diff
@@ -0,0 +1 @@
+trial --help-reactors will now display iocp and win32er reactors with Python 3.
```

Comment:
Note `twistd` also has a `--help-reactors` option and this fixes that as well.

Also, did  you mean to have a bugfix and a feature news fragment for this change?  It seems like one or the other is enough to draw user attention to it... But if you think the iocp/win32er aspect of the change is significant enough to merit special attention, that's okay with me.